### ISSUE TITLE
[Issue]: Enabling fMP4-HLS in Jellyfin settings causes issues with subtitles and media."

### ISSUE THREAD

**Issue by Kochisa** [jellyfin/jellyfin#10273]
### Please describe your bug

When playing videos with audio encoded in formats like DTS and EAC3, subtitles are completely out of sync. This issue does not occur when fMP4-HLS is not used.

### Jellyfin Version

10.8.z

### Environment

```markdown
- OS:Windows 11 Professional Workstation Edition
- Browser:Chrome
- FFmpeg Version:5.1.2-Jellyfin
- Playback Method:play directly
- Hardware Acceleration:None
- GPU Model:Intel 6700K
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by felix920506** [comment]
Please submit ffmpeg logs.

**Comment by gnattu** [comment]
Should already be fixed in master(10.9)

**Comment by gnattu** [comment]
If you see this happening again in 10.9, please open a new issue instead of replying in this one. Thank you.
